tests/cypress/tests/e2e/mcpar/form.feature

```diff
@@ -3,7 +3,13 @@ Feature: MCPAR E2E Form Submission
     Scenario: A state user can fully create a form and submit it
         Given I am logged in as a state user
         When I submit a new MCPAR program
-        Then the program is submitted
+# Then the program is submitted
+
```

Comment:
this belongs to another story.

Reply:
Meaning another stepdef file? I.e `submit.stepdef.js`

Reply:
was submit part of this story? I saw the 'incomplete' program and (perhaps wrongly) assumed it was accidentally blended from statusing.

Reply:
No, I needed the submit handler in this branch so I cherry-picked my commit from `statusing`

Reply:
Long story short—MLR has unique behavior on submit. Rather than add that to an update flow which we were deprecating anyways, I moved the submit handler (and associated tests) over.